demo/pages/index.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import NewComponent from '@moxy/react-lib-template';
+import ContentfulImage from '@moxy/react-contentful-image';
 
 import styles from './index.module.css';
 
 const Home = () => (
-    <div>
-        <h1>react-lib-template</h1>
-        <NewComponent className={ styles.newComponent }>Hello World</NewComponent>
+    <div className={ styles.container }>
+        <h1>Contentful Image</h1>
+        <ContentfulImage
+            backgroundColor="#9090ff"
+            resize={ { width: 100, height: 100, behavior: 'pad' } }
+            src="//images.ctfassets.net/yadj1kx9rmg0/wtrHxeu3zEoEce2MokCSi/cf6f68efdcf625fdc060607df0f3baef/quwowooybuqbl6ntboz3.jpg" />
```

Comment:
Again, does this work with the `//`? 🤔 

I remember having some trouble with that when fetching images from Contentful.

Reply:
I've tested this and it works so I'm probably confusing this with some other issue I had, so all good!

Reply:
Yap, you actually need them in order to get the image properly.